src/comment-modal.js

```diff
@@ -0,0 +1,106 @@
+import { commentPost, getLastTwoComments } from './comment-form.js';
+import calCom from './comment-counter.js';
+
+function createCommentModal(product) {
+  const commentModal = document.createElement('div');
+  commentModal.classList.add('comment-modal');
+  commentModal.classList.add('hidden');
+
+  const closeBtn = document.createElement('i');
+  closeBtn.classList.add('fas', 'fa-times');
+  commentModal.appendChild(closeBtn);
+
+  closeBtn.addEventListener('click', () => {
+    commentModal.style.display = 'none';
+  });
```

Comment:
- [ ] Good job so far, but I think you still have some issues that need to be fixed. When opening the comment popup it appears and I can add a comment and do whatever I want but if I close the comment popup it can't open again when I click on the comment button. Users should be able to open the comment popup window at any time, even after closing it they can reopen it and it should open normally, you should fix that 🙃 🙃

https://user-images.githubusercontent.com/74501523/219637012-5d5479ac-d5dd-4194-a476-9a1699cc7a36.mov

- [ ] Also your deployed app misses the logo; the image doesn’t show. Please kindly fix that, you should make sure that your deployed app work correctly, and everything like image, and links,... works correctly 🙃 😉 

<img width="795" alt="Screen Shot 2023-02-17 at 14 13 26" src="https://user-images.githubusercontent.com/74501523/219637902-49753a85-3f6e-443a-b413-14ac926347e3.png">